cypress/integration/listing-template.js

```diff
@@ -29,13 +29,11 @@ if (Cypress.env('API') !== 'guillotina') {
       cy.get('#toolbar-add').click();
       cy.get('#toolbar-add-image').click();
       cy.get('input[name="title"]').type('My image');
-      cy.fixture('image.png', 'base64').then(fileContent => {
-        cy.get('#field-image').upload(
-          { fileContent, fileName: 'image.png', mimeType: 'image/png' },
-          { subjectType: 'input' },
-        );
+      cy.get('#field-image').attachFile('image.png', {
+        subjectType: 'input',
       });
       cy.get('#toolbar-save').click();
+      cy.wait(4000);
```

Comment:
@iFlameing it would be better to use cy.waitForResourceToLoad here instead of a wait. A wait slows down the tests while a waitForResourceTo load only waits as long as necessary.